tests/Test.MSAL.NET.Unit/TelemetryTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿//----------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Identity.Client;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Test.MSAL.NET.Unit
+{
+    class MyReceiver : IReceiver
+    {
+        public void onEvents(List<Dictionary<string, string>> events)
+        {
+            foreach(Dictionary<string, string> e in events)
+            {
+                foreach(KeyValuePair<string, string> entry in e)
+                {
+                    Console.WriteLine("{0}: {1}", entry.Key, entry.Value);
+                }
+            }
+        }
+    }
+
+    [TestClass]
+    public class TelemetryTests
+    {
+        [TestInitialize]
+        public void Initialize()
+        {
+        }
+
+        [TestMethod]
+        public void TelemetryPublicApiSample()
+        {
+            Telemetry telemetry = Telemetry.GetInstance();
+            MyReceiver receiver = new MyReceiver();
+            telemetry.RegisterReceiver(receiver);
+
+            // Or you can use a one-liner:
+            Telemetry.GetInstance().RegisterReceiver(new MyReceiver());
```

Comment:
I would suggest to keep this `TelemetryPublicApiSample` as-is, in order to demonstrate how to use the Telemetry, without any noise. I would add a separated test case to assert telemetry is not null and assert it is a singleton.